ipv-core/src/main/java/com/ibm/whc/deid/util/HashUtils.java

```diff
@@ -5,33 +5,20 @@
  */
 package com.ibm.whc.deid.util;
 
-import java.math.BigInteger;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
-import com.ibm.whc.deid.utils.log.LogCodes;
-import com.ibm.whc.deid.utils.log.LogManager;
+import java.nio.charset.Charset;
+import java.util.UUID;
+
 
 public class HashUtils {
-  private static final LogManager logger = LogManager.getInstance();
 
   public static Long longFromHash(String value) {
-    return longFromHash(value, "SHA-256");
-  }
-
-  public static Long longFromHash(String value, String algorithm) {
-    try {
-      MessageDigest md = MessageDigest.getInstance(algorithm);
-
-      if (value == null) {
-        value = "";
-      }
+    if (value == null) {
+      value = "";
+    }
 
-      md.update(value.getBytes());
-      return new BigInteger(md.digest()).longValue();
-    } catch (NoSuchAlgorithmException e) {
-      logger.logError(LogCodes.WPH1013E, e);
+    long hash =
+        UUID.nameUUIDFromBytes(value.getBytes(Charset.forName("UTF-8"))).getMostSignificantBits();
+    return hash;
```

Comment:
I sugest changing the method signature to return "long" instead of "Long".  Callers might be expecting "Long", but the response should get boxed.  Since the response can't be null, "long" seem like a more appropriate type.

Reply:
Changed to long as suggested.